mgradm/cmd/install/podman/utils.go

```diff
@@ -67,16 +68,27 @@ func installForPodman(
 	cmd *cobra.Command,
 	args []string,
 ) error {
-	flags.CheckParameters(cmd, "podman")
-	if _, err := exec.LookPath("podman"); err != nil {
-		return errors.New(L("install podman before running this command"))
-	}
-
 	inspectedHostValues, err := utils.InspectHost(false)
 	if err != nil {
 		return utils.Errorf(err, L("cannot inspect host values"))
 	}
 
+	/* Detect existing installation
+	- running server or configured, but not running server -> fail
+	- unconfigured, but volumes exists -> warn and continue
+	*/
+	if _, present := inspectedHostValues["host_uyuni_server"]; present {
+		return fmt.Errorf(L("Server is already initialized! Uninstall before attempting new installation or use upgrade command"))
+	}
+	if _, present := inspectedHostValues["host_uyuni_volume"]; present {
+		log.Warn().Msgf(L("Server volume is already present. Installation will continue, but installation options may not match. Make sure entered passwords match."))
```

Comment:
Aren't the passwords simply ignored in such a case?
```suggestion
		log.Warn().Msgf(L("Server volume is already present. Installation will continue, but installation options may be ignored."))
```

Reply:
I am using the login and password to check if the first organization already exists and we are able to login.

Reply:
> I am using the login and password to check if the first organization already exists and we are able to login.

Then may be this should be more explicit in the message. It starts to be quite long, we may have to cut it in two lines

```suggestion
		log.Warn().Msgf(L("Server volume is already present. Installation will continue, but installation options may be ignored. Make sure the administrator username and passwords match the previously set ones."))
```

Reply:
Do you think is needed to check the password are the same? Isn't enough just a warning saying the provided data will be ignored?